openapi/p4pa-pu-bff.openapi.yaml

```
@@ -1652,6 +1652,41 @@ paths:
         '500':
           description: Internal Server Error
       x-spring-paginated: true
+  /organization/{organizationId}/printpaymentnotice/generate/{debtPositionId}:
+    post:
+      tags:
+        - print-payment-notice
```

This is still a DebtPosition operation
```suggestion
        - DebtPositions
```